packages/react-component-library/package.json

```diff
@@ -50,9 +50,6 @@
   "lint-staged": {
     "*.@(js|jsx|ts|tsx)": "eslint"
   },
-  "jest": {
-    "testResultsProcessor": "jest-sonar-reporter"
```

Comment:
Not needed as it's specified here:

https://github.com/defencedigital/mod-uk-design-system/blob/5b581d7c4b0e30fbedb2a088da3b17e559429848/.github/workflows/build_and_test_feature.yml#L152